### PR TITLE
CHF-356 : Modifying Readme for Core Devices

### PR DESCRIPTION
**Pull request by skt123**
Added Readme files for the following devices:
1. OSRAM Lightify LED On/Off/Dim
2. OSRAM Lightify LED RGBW
3. OSRAM Lightify Tunable 60 White

Modified Readme files for the following devices:
1. SmartSense Moisture Sensor
2. SmartSense Temp/Humidity Sensor
3. SmartSense Multi Sensor
4. SmartSense Open/closed Sensor
5. SmartPower Outlet
6. Connected Cree Bulb
7. SmartSense Motion Sensor

### CODE REVIEW

**Comment by skt123** [comment]
@ShunmugaSundar @jackchi 
Please review the changes and merge.

**Comment by skt123** [comment]
@ShunmugaSundar @jackchi 
Please review the changes and merge.

File: devicetypes/smartthings/cree-bulb.src/README.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ Works with:
 
 ## Device Health
 
-A Category C6 Connected Cree LED Bulb with maxReportTime of 10 min.
-Check-in interval is double the value of maxReportTime for Zigbee device. 
-This gives the device twice the amount of time to respond before it is marked as offline.
-Check-in interval = 2*10 = 20 min
+A Category C6 Connected Cree LED Bulb with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime.
```
Comment (jackchi) [inline devicetypes/smartthings/cree-bulb.src/README.md]: We promised VD that devices going offline shall be notified within `16min`. To pick a check-in interval that works we went with `12min`, which gives the device 2 chances to check in before the ping. 
Let's change this to `2x` and `12min`'

Actually the `cree-bulb` doesn't periodically check in. Which is why we needed to implement the scheduled() reading attributes. Currently this is not in Master yet, I'll be submitting a PR soon.

File: devicetypes/smartthings/smartpower-outlet.src/README.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ Works with:
...
 ## Device Health
 
-A Category C1 smart power outlet with maxReportTime of 10 min.
-Check-in interval is double the value of maxReportTime for Zigbee device. 
-This gives the device twice the amount of time to respond before it is marked as offline.
-Check-in interval = 2*10 = 20 min
+A Category C1 smart power outlet with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/smartpower-outlet.src/README.md]: Same here

File: devicetypes/smartthings/smartsense-moisture-sensor.src/README.md [hunk 3]
```diff
@@ -23,10 +23,10 @@ Works with:
...
 ## Device Health
 
-A Category C2 moisture sensor with maxReportTime of 1 hr.
-Check-in interval is double the value of maxReportTime for Zigbee device. 
-This gives the device twice the amount of time to respond before it is marked as offline.
-Check-in interval = 2*60 = 120 min
+A Category C2 moisture sensor with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/README.md]: Same here

File: devicetypes/smartthings/smartsense-motion-sensor.src/README.md [hunk 4]
```diff
@@ -22,10 +22,10 @@ Works with:
...
 ## Device Health
 
-A Category C2 motion sensor with maxReportTime of 1 hr.
-Check-in interval is double the value of maxReportTime for Zigbee device. 
-This gives the device twice the amount of time to respond before it is marked as offline.
-Check-in interval = 2*60 = 120 min
+A Category C2 motion sensor with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime.
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/smartsense-motion-sensor.src/README.md]: Same here

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/README.md [hunk 5]
```diff
@@ -24,10 +24,10 @@ Works with:
...
 ## Device Health
 
-A Category C2 open/closed sensor with maxReportTime of 1 hr.
-Check-in interval is double the value of maxReportTime for Zigbee device. 
-This gives the device twice the amount of time to respond before it is marked as offline.
-Check-in interval = 2*60 = 120 min
+A Category C2 open/closed sensor with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/README.md]: Same here

File: devicetypes/smartthings/smartsense-temp-humidity-sensor.src/README.md [hunk 6]
```diff
@@ -24,10 +24,10 @@ Works with:
...
 ## Device Health
 
-A Category C2 SmartSense Temp/Humidity Sensor with maxReportTime of 1 hr.
-Check-in interval is double the value of maxReportTime for Zigbee device. 
-This gives the device twice the amount of time to respond before it is marked as offline.
-Check-in interval = 2*60 = 120 min
+A Category C2 SmartSense Temp/Humidity Sensor with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/smartsense-temp-humidity-sensor.src/README.md]: Same here

File: devicetypes/smartthings/zigbee-dimmer.src/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* **Switch** - can detect state (possible values: on/off)
+* **Switch Level** - represents current light level, usually 0-100 in percent
+* **Health Check** - indicates ability to get device health notifications
+
+## Device Health
+
+A Category C1 Zigbee dimmer with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/zigbee-dimmer.src/README.md]: Same here

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/README.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+* **Refresh** - _refresh()_ command for status updates
+* **Health Check** - indicates ability to get device health notifications
+
+
+## Device Health
+
+A Category C6 OSRAM LIGHTIFY LED RGBW Bulb with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/README.md]: Same here

File: devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/README.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+* **Switch** - can detect state (possible values: on/off)
+* **Switch Level** - represents current light level, usually 0-100 in percent
+* **Health Check** - indicates ability to get device health notifications
+
+## Device Health
+
+A Category C1 OSRAM Lightify Tunable 60 White with maxReportTime of 5 mins.
+Check-in interval is three times the value of maxReportTime. 
+This gives the device thrice the amount of time to respond before it is marked as offline.
+Check-in interval = 3*5 = 15 mins
```
Comment (jackchi) [inline devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/README.md]: Same here

**Comment by jackchi** [comment]
👍 Looks good!

**Pull request merged**